### PR TITLE
Portal V2 UI

### PR DESCRIPTION
**Pull request by sudheshna-donthineni**

### CODE REVIEW

File: src/components/Dropdown.vue [hunk 1]
```diff
@@ -1,18 +1,31 @@
...
       type="select"
       v-model="value"
       :options="options"
-      placeholder="Select"
-      :validation="[isRequired ? ['required'] : []]"
-      validation-visibility="dirty"
       :name="dbKey"
       :help="helpText"
+      placeholder=" "
+      :input-class="{ 'w-full': true }"
```
Comment (suryabulusu) [inline src/components/Dropdown.vue:14]: can simplify as 
```:inner-class="{
  border: true,
  'py-2 px-2 rounded border-grey overflow-hidden': true,
}"
:help-class="{
  'mt-[10px] text-sm text-grey italic': true,
}"```

File: src/components/Dropdown.vue [hunk 2]
```diff
@@ -50,13 +63,24 @@ export default {
       type: Boolean,
       default: false,
     },
+    defaultValue: {
```
Comment (suryabulusu) [inline src/components/Dropdown.vue]: using this where?

File: src/components/Dropdown.vue [hunk 3]
```diff
@@ -50,13 +63,24 @@ export default {
...
+      type: String,
+      default: "",
+    },
+    invalid: {
+      type: Boolean,
+      default: false,
+    },
   },
   data() {
```
Comment (suryabulusu) [inline src/components/Dropdown.vue]: what does this do?

File: src/components/NewPhoneNumberEntry.vue [hunk 4]
```diff
@@ -1,24 +1,40 @@
...
       v-model="phoneNumber"
       @keypress="isValidPhoneNumberEntry($event)"
       @input="updatePhoneNumberEntry($event)"
       ondrop="return false"
       onpaste="return false"
       :help="helpText"
+      :wrapper-class="{
+        'mx-auto': true,
+      }"
+      :inner-class="{
```
Comment (suryabulusu) [inline src/components/NewPhoneNumberEntry.vue:18]: :inner-class="{
  border: true,
  'py-2 px-2 rounded mx-auto': true,
  'border-red': this.invalid,
  'border-grey': !this.invalid,
}"

File: src/pages/InformationForm.vue [hunk 5]
```diff
@@ -98,29 +92,39 @@ export default {
       this.formSchemaData[field]["required"] =
         this.formSchemaData[field].required == "TRUE" ? true : false;
     });
+    this.isUserDataIsComplete();
```
Comment (suryabulusu) [inline src/pages/InformationForm.vue:95]: isUserDataComplete()

File: src/components/LanguagePicker.vue [hunk 6]
```diff
@@ -1,28 +1,39 @@
...
+    };
+  },
+  mounted() {
+    this.language = this.$store.state.language;
+  },
+  watch: {
+    /**
+     * Sets the language in the store based on the user input
+     */
+    language() {
```
Comment (suryabulusu) [inline src/components/LanguagePicker.vue]: call it locale instead of language like other parts of code?

File: src/pages/InformationForm.vue (hunk 5 above)
Comment (suryabulusu) [reply src/pages/InformationForm.vue:95]: ah there is a variable like this. maybe call it checkIfUserDataIsComplete() then?

File: src/pages/NewSignup.vue [hunk 7]
```diff
@@ -39,36 +39,53 @@
...
+      class="bg-primary-hover py-10 rounded-md shadow-sm"
+    >
       <p>
-        Your ID is <b>{{ userData["user_id"] }}</b>
+        <b>Your ID is {{ userData["user_id"] }}</b>
       </p>
-      <p>Please note this down. Use this to sign-in going forward.</p>
-    </template>
+      <p>
+        Kindly make a note of it. You will need this to log in to all your
```
Comment (suryabulusu) [inline src/pages/NewSignup.vue:74]: wouldnt this require translations too

File: src/pages/NewSignup.vue [hunk 8]
```diff
@@ -113,16 +130,35 @@ export default {
...
   },
   computed: {
+    /** Returns button text */
+    signUpButtonLabel() {
+      return this.getLocale == "en" ? "Sign Up" : "साइन अप";
+    },
+
+    /** Returns button text */
+    startSessionText() {
+      return this.getLocale == "en" ? "Start Session" : "सत्र शुरू करें";
```
Comment (suryabulusu) [inline src/pages/NewSignup.vue:144]: call it "session" in hindi instead of satra 😭

**Review by suryabulusu — COMMENTED** [review]
left comments

File: src/pages/NewSignup.vue (hunk 8 above)
Comment (sudheshna-donthineni) [reply src/pages/NewSignup.vue:144]: Unfortunately, the tech team is not responsible for the translations. We get it from ops.

File: src/components/Dropdown.vue (hunk 2 above)
Comment (sudheshna-donthineni) [reply src/components/Dropdown.vue]: Oh, I was testing something. Forgot to remove it.

File: src/components/Dropdown.vue (hunk 3 above)
Comment (sudheshna-donthineni) [reply src/components/Dropdown.vue]: Same here. Forgot to remove it.

File: src/pages/InformationForm.vue (hunk 5 above)
Comment (sudheshna-donthineni) [reply src/pages/InformationForm.vue:95]: its just a local variable no?

File: src/components/Dropdown.vue (hunk 1 above)
Comment (sudheshna-donthineni) [reply src/components/Dropdown.vue:14]: yeah, makes sense.

File: src/pages/NewSignup.vue (hunk 7 above)
Comment (sudheshna-donthineni) [reply src/pages/NewSignup.vue:74]: Haven't gotten it. When I do, I'll add it.

**Pull request merged**